src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -1072,7 +1071,7 @@ instruct vneg(vReg dst, vReg src) %{
   match(Set dst (NegVI src));
   match(Set dst (NegVL src));
   ins_cost(VEC_COST);
-  format %{ "vrsub.vx $dst, $src, $src\t#@vneg" %}
+  format %{ "vneg $dst, $src, $src" %}
```

Comment:
Hi, for vneg and vfneg, I think `vneg $dst, $src` is better.

Reply:
Hi, Thanks for your review. and has been fixed.